src/main/webapp/app/components/confirm-button/confirm-button.component.ts

```diff
@@ -1,7 +1,8 @@
 import { Component, EventEmitter, OnInit, Output } from '@angular/core';
 
+// TODO this can probably be moved to one of our generic delete components
 @Component({
-    selector: 'jhi-confirm-button',
+    selector: 'jhi-confirm-delete-button',
```

Comment:
why did you change this?

Reply:
Functionality-wise this was a delete button. It had "Delete" as the string and a confirmation, before there was confirmation event fired. We should use the new generic delete component in this case anyway, so I now completely removed this component (we only used it for deleting lecture attachments) and switched the usage in related components to our new `jhiDeleteButton` directive